### PR TITLE
contributors: add sre-bot command help doc

### PR DESCRIPTION
**Pull request by zz-jason**
as the title says.

### CODE REVIEW

File: contributors/command-help.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+## Continuous Integration(CI) Commands
+
+Supported Repositories:
+
+* [tidb](https://github.com/pingcap/tidb)
+* [tikv](https://github.com/tikv/tikv)
+* [pd](https://github.com/pingcap/pd)
+* [tispark](https://github.com/pingcap/tispark)
+* [tidb-binlog](https://github.com/pingcap/tidb-binlog)
+* [tidb-test](https://github.com/pingcap/tidb-test)
```
Comment (zhouqiang-cl) [inline contributors/command-help.md]: tidb-test is private repo

File: contributors/command-help.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+Run a single CI test. All the supported tests are:
+
+```
+/run-common-test
+/run-integration-common-test
+/run-integration-compatibility-test
+/run-integration-ddl-test
+/run-mybatis-test
+/run-sqllogic-test-1
```
Comment (zhouqiang-cl) [inline contributors/command-help.md]: some repo did not have this test

File: contributors/command-help.md (hunk 1 above)
Comment (zz-jason) [reply contributors/command-help.md]: removed.

File: contributors/command-help.md (hunk 2 above)
Comment (zz-jason) [reply contributors/command-help.md]: I've added an explanation on the following `NOTE` section.

**Comment by gingerkidney** [comment]
LGTM

**Pull request merged**